demo/demo.md

```diff
@@ -0,0 +1,117 @@
+  <!-- The below content is automatically added from ./../../apiExamples/basic.html -->
+  <auro-pane date="2020-09-20"></auro-pane>
+  <auro-pane date="2020-09-21" selected></auro-pane>
+  <auro-pane date="2020-09-22" disabled></auro-pane>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/basic.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/basic.html -->
+
+```html
+<auro-pane date="2020-09-20"></auro-pane>
+<auro-pane date="2020-09-21" selected></auro-pane>
+<auro-pane date="2020-09-22" disabled></auro-pane>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+### Small
+
+<div class="exampleWrapper">
+  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/small.html) -->
+  <!-- The below content is automatically added from ./../../apiExamples/small.html -->
+  <auro-pane date="2020-09-20" price="$500" sm></auro-pane>
+  <auro-pane date="2020-09-21" price="$501" selected="true" sm></auro-pane>
+  <auro-pane date="2020-09-22" price="$480" disabled="true" sm></auro-pane>
+  <!-- AURO-GENERATED-CONTENT:END -->
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/small.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/small.html -->
+
+```html
+<auro-pane date="2020-09-20" price="$500" sm></auro-pane>
+<auro-pane date="2020-09-21" price="$501" selected="true" sm></auro-pane>
+<auro-pane date="2020-09-22" price="$480" disabled="true" sm></auro-pane>
+```
+<!-- AURO-GENERATED-CONTENT:END -->
+</auro-accordion>
+
+## Recommended Use and Version Control
+
+There are two important parts of every Auro component. The <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes">class</a> and the custom clement. The class is exported and then used as part of defining the Web Component. When importing this component as described in the <a href="#install">install</a> section, the class is imported and the `auro-pane` custom element is defined automatically.
+
+To protect from versioning conflicts with other instances of the component being loaded, it is recommended to use our `registerComponent(name)` method and pass in a unique name.
+
+```js
+import './node_modules/@alaskaairux/auro-pane';
+registerComponent('custom-pane');
+```
+
+This will create a new custom element that you can use in your HTML that will function identically to the `auro-pane` element.
+
+<div class="exampleWrapper">
+  <custom-header display="display">Salutations World!</custom-header>
+</div>
+<auro-accordion lowProfile justifyRight>
+  <span slot="trigger">See code</span>
+  ```html
+  <custom-header display="display">Salutations World!</custom-header>
+  ```
+
+</auro-accordion>
```

Comment:
This example is not working in the displayed page. 

<img width="1089" alt="Screen Shot 2023-08-17 at 6 05 13 PM" src="https://github.com/AlaskaAirlines/auro-pane/assets/181089/bd96a5e8-34b3-4c2d-a8cd-38703764d299">